checks/system/network/cxi_stat_hsn.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class CXIStatHSN(rfm.RunOnlyRegressionTest):
+    valid_systems = ['+nvgpu']
+    valid_prog_environs = ['builtin']
```

Comment:
```suggestion
    valid_prog_environs = ['+cuda']
```
Not 100% sure but that would allow UENV:
```
[ DRY      ] CXIStatHSN /6b50cbc5 @todi:normal+prgenv-gnu_24.7_v3_default
```
and CPE:
```
[ DRY      ] CXIStatHSN /6b50cbc5 @todi:normal+PrgEnv-cray
[ DRY      ] CXIStatHSN /6b50cbc5 @todi:normal+PrgEnv-gnu
```
because builtin is not set in reframe.yaml, for example with `prgenv-gnu/24.7:v3`:
```
default:
  features:
    - cuda
    - mpi
    - osu-micro-benchmarks
    - openmp
    - serial
```

Reply:
All those are system tests so they don't test a particular software package. The cxi utilities come with the system.